pkg/sypgp/sypgp.go

```diff
@@ -564,6 +564,9 @@ func EncryptKey(k *openpgp.Entity, pass string) error {
 
 // SelectPubKey prints a public key list to user and returns the choice
 func SelectPubKey(el openpgp.EntityList) (*openpgp.Entity, error) {
+	if el == nil {
+		return nil, fmt.Errorf("no key in public keyring")
```

Comment:
maybe add a separate error for that cases? e.g `ErrEmptyKeyList`
also you can check if a list is empty with `len(el)` to eliminate case where slice is not nil but still empty.

Reply:
Good idea, I will need to use two variables, `errPublicKeyringEmpty`, and `errPrivateKeyringEmpty` for the public and private keyring.

Reply:
I think one exported would be enough, the context will be clear depending on function being called